features/201.Introduction_to_IoT_Sensors.feature

```diff
@@ -14,10 +14,11 @@ Feature: test tutorial 201.Introduction to IoT Sensors
 
 
     Scenario Outline: Communicating with IoT Devices: Using Actuators
-    When  I send POST HTTP IoT request for "<description>" to "http://localhost:3001"/"<location>"
-    And   With the body IoT request described in "<request_file>"
-    Then  I receive a HTTP "200" IoT response code with the body "<response_file>"
-
+    When  I prepare a POST HTTP request for "<description>" to "http://localhost:3001/<location>"
+    And   I set header Content-Type to application/x-www-form-urlencoded
```

Comment:
Take a look to the implementation in 402 feature:

@when('I set the "{key}" header with the value "{value}"')

Reply:
402 feature wasn't there when this was implemented. I'll refactor in the future.

Reply:
NTC